### DISCUSSION TITLE
Meteor run android issue

### DISCUSSION THREAD

**Discussion by problemsolver-eng** [community/community#124078] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

#Meteor run android issue:

I am tyring to build app using Meteor:

But I get the following error:

Error: Command failed: ...\.meteor\local\cordova-build\platforms\android\cordova\version
'...\.meteor\local\cordova-build\platforms\android\cordova\version' is not recognized as an internal or external command,
operable program or batch file.

**Comment by brandonsoft** [comment]
It's caused by cordova removing the version.bat file for some reason, it has been reported here: https://github.com/apache/cordova-android/issues/1392

But looks like not everyone has this problem
If you want to temporarily fix this issue, place this code in .meteor\local\cordova-build\platforms\android\cordova\version.bat

```
:: Licensed to the Apache Software Foundation (ASF) under one
:: or more contributor license agreements.  See the NOTICE file
:: distributed with this work for additional information
:: regarding copyright ownership.  The ASF licenses this file
:: to you under the Apache License, Version 2.0 (the
:: "License"); you may not use this file except in compliance
:: with the License.  You may obtain a copy of the License at
::
:: http://www.apache.org/licenses/LICENSE-2.0
::
:: Unless required by applicable law or agreed to in writing,
:: software distributed under the License is distributed on an
:: "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
:: KIND, either express or implied.  See the License for the
:: specific language governing permissions and limitations
:: under the License.

@ECHO OFF
SET script_path="%~dp0version"
IF EXIST %script_path% (
        node %script_path% %*
) ELSE (
    ECHO.
    ECHO ERROR: Could not find 'version' script in 'cordova' folder, aborting...>&2
    EXIT /B 1
)
```

**Comment by ebndev** [comment]
Hi @dreambuilder00 & @awesomedev77, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
